chat-commands.js

```diff
@@ -2292,21 +2294,21 @@ exports.commands = {
 			// Get a list of all the rooms
 			let fileList = fs.readdirSync('logs/modlog');
 			for (let i = 0; i < fileList.length; ++i) {
-				filename += path.normalize(__dirname + '/' + logPath + fileList[i]) + ' ';
+				filename += path.normalize(`${__dirname}/${logPath}${fileList[i]}`) + ' ';
```

Comment:
About the same. I think it's plenty readable with template strings. The bigger problem is `path.normalize` here wrecking readability. :/